app/ui-react/packages/models/src/extra.d.ts

```diff
@@ -53,6 +53,34 @@ export interface IApiSummarySoap extends APISummary {
   connectorTemplateId?: string;
 }
 
+/**
+ * Customizable properties in API Client Connector wizard
+ */
+export interface ICreateConnectorProps {
```

Comment:
I'm on the fence about putting this interface here.  Is this specific to a component or API function?  Maybe it should stay alongside the function/component where it's used.  Usually the `extra.d.ts` file is for creating "massaged" API types vs. component property interfaces.

Reply:
It's currently used in `@syndesis SecurityPage,  DetailsPage, ReviewActionsPage` and `@api useApiConnectorCreator`. There was need for a very similar interface in the `@ui` package as well, but we can't use `@models` in there, so I created `ICreateConnectorPropsUi`. I was trying to avoid was duplicating code that is indirectly based on an existing model, but I'm happy to provide different interfaces for the api, syndesis, and UI packages if you think that's the best thing, and keeping this interface where it was. Let me know what you think?

Reply:
Yeah, I'd keep separate models for the UI component, even if you create a `models.ts` if it's shared by a few files.